examples/bigcommerce-catalyst/next.config.js

```diff
@@ -0,0 +1,29 @@
+// @ts-check
+const withMakeswift = require('@makeswift/runtime/next/plugin')();
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  images: {
+    remotePatterns: [
+      {
+        hostname: process.env.BIGCOMMERCE_CDN_HOSTNAME ?? '*.bigcommerce.com',
+      },
+    ],
+  },
+  transpilePackages: ['@bigcommerce/components'],
+  typescript: {
+    ignoreBuildErrors: !!process.env.CI,
+  },
+  eslint: {
+    ignoreDuringBuilds: !!process.env.CI,
+    dirs: ['app', 'client', 'components', 'lib', 'middlewares', 'pages', 'providers'],
+  },
+  // default URL generation in BigCommerce uses trailing slash
+  trailingSlash: process.env.TRAILING_SLASH !== 'false',
+  experimental: {
+    optimizePackageImports: ['@icons-pack/react-simple-icons'],
+  },
```

Comment:
This is a notable change. Basically, we were running into [this issue](https://github.com/phosphor-icons/react/issues/45). I believe this should now build and run.